CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 3.11.0 (December 16, 2025)
+
+BUG FIXES:
+* `resource/nsxt_policy_bgp_neighbor`: Fix maximum password length to match NSX spec ([#1870](https://github.com/vmware/terraform-provider-nsxt/pull/1870))
+* `resource/nsxt_policy_tier0_gateway`: Fix inter_sr_ibgp value check issue ([#1877](https://github.com/vmware/terraform-provider-nsxt/pull/1877))
+* `resource/nsxt_policy_predefined_gateway_policy`: Ignore gateway policy default rule order ([#1878](https://github.com/vmware/terraform-provider-nsxt/pull/1878))
+* `resource/nsxt_policy_vlan_segment`: Interface conversion error when applying config to VLAN Backed ([#1879](https://github.com/vmware/terraform-provider-nsxt/pull/1879))
+* `resource/nsxt_policy_tier0_gateway_interface`: Read edge_cluster_member_index from the state ([#1880](https://github.com/vmware/terraform-provider-nsxt/pull/1880))
+* `resource/nsxt_policy_lb_virtual_server`: Add a check to identify import operation in read function ([#1881](https://github.com/vmware/terraform-provider-nsxt/pull/1881))
```

Comment:
Can you rephrase this "add a check to ensure ensure LB rules for virtual servers are imported"